src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+}
+
+@for $i from 1 through 5 {
+  .stars--#{$i} > .stars__star:nth-child(-n + #{$i}) {
+    background-image: url('../images/star-active.svg');
```

Comment:
The path to the image should use double quotes, not single quotes, following the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). It should be `url("../images/star-active.svg")`.